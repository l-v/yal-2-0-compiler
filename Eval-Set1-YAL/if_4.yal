module if_4 {

function m=max(a,b) {
		if(a > b) {
			m = a;
		} else {
			m = b;
		}
}

function m=min(a,b) {
		if(a > b) {
			m = b;
		} else {
			m = a;
		}
}

function main() {
  d = 1;
  e = 2;
  m1 = max(d, e);
  io.println("m1: ",m1); // 2

  m1 = min(d, e);
  io.println("m1: ",m1); // 1
}
}

