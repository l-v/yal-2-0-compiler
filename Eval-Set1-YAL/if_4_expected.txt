m1: 2
m1: 1
